bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/nici/Test.java

```diff
@@ -73,8 +74,12 @@ public static void main(String[] argv) throws IOException {
                     new ObservingConditionParameters(2, 2, 4, 3, 2.3);
             
             NiciParameters np =
-                    new NiciParameters("R",
-                    NiciParameters.NDA);
+                    new NiciParameters(
+                        "H",                    // channel1
+                        "ndfilt_clear",         // channel2
+                        "80",                   // pupil
+                        "",                     // instrument mode, unused?
+                        "h5050");               // dichroic position
```

Comment:
Yes absolutely. I would like to replace as many of those strings as possible by the types from the OCS core and instruments. I might have to make some compromises depending on available time etc. But as long as nobody stops me I am planning to clean this up as much as possible. This is also the main reason why I was pushing for integrating the ITC into the OCS source tree so that I can use that code.